0015 Community Membership Process.md

```diff
@@ -0,0 +1,83 @@
+# Process for Defining Community Member Roles  
+
+* Status: proposed
+
+* Input: Welcome from anyone!
+* Deciders:  * Ezra and anyone with merge rights on any repo.
+
+* Date: {2022/07/24 last updated} <!-- optional -->
+
+## Context and Problem Statement
+
+Leadership and Team Structure has been more or less ad hoc. There is a need for a community-driven process to:
+1. Define and recognize needed positions
+2. Formalize who is holding responsibility for a specific position.
+3. Provide new opportunities
+4. Clearly define teams
+5. Replace inactive team members with positions
+
+## Decision Drivers <!-- optional -->
+
+* Functionality - will the new process work and gain buy-in from the community.
+* Scalability - will the process allow for a constant influx of new talent
+* Maintainability - will the process be easy to maintain or self-maintaining
+
+## Considered Options
+
+* Option #1 - Direct new-comers and team members to edit a GitHub team page such as this example: https://github.com/Greenstand/START-Greenstand-overview/blob/master/TEAM.md
+
+This page can function as:
+1. A listing for available positions
+2. A listing of leadership roles
+3. Identifying team members
+4. Place for leads to list and contributors to accept micro roles
+4. A first pull request for novice developers
+
+* Option #2 - Continued listing on the website updated by an HR contributor.
+
+* Option #3 - Your suggestions are welcome
+
+
+## Decision Outcome
+
+To be decided
+
+### Positive Consequences <!-- optional -->
+
+Option #1
+* Improve the development of the platform through clear delegation of leadership roles and responsibilities.
+* Help prevent burn-out by enabling people to vote others into small roles and responsibilities
+* Allows people to self onboard and creates team engagement
+
+### Negative Consequences <!-- optional -->
+
+Option #1
+* It may feel non-inclusive for non-github users
+
+## Pros and Cons of the Options <!-- optional -->
+
+### {option 1}
+
+* Good, because it is easy to implement
+* Good, because it is community-driven
+* Good, because it can represent current teams
+* Good, because it is a more organic process
+* Bad, maintaining it is extra energy, but any system requires this
+* Bad, because requires Maintenance.
+* Bad, Github is not easy for operations teams to use
+
+### {option 2}
+
+* Good, because it is a more common workflow for HR Related tasks
+* Bad, because it requires more top-down desicions and may not capture actual needs.
+* Bad, because current work flows have lead to general ambiguity about who is doing what.
+* Bad, because it has a lot of room for improvement
```

Comment:
Maybe we can have them both, option1 and 2, by publishing the team.md to our website?

There are tools like this: https://pages.github.com/

Reply:
So in this way, using Github, we can manage the workflow of organizing and building our team, and by publishing, we get the formal announcement page of the team, on our website